packages/vue/src/components/molecules/SfSelect/SfSelect.vue

```diff
@@ -13,11 +13,11 @@
     }"
     class="sf-select"
     @click="toggle($event)"
-    @blur="closeHandler"
     @keyup.space="openHandler"
     @keyup.up="move(-1)"
     @keyup.down="move(1)"
     @keyup.enter="enter($event)"
+    v-clickOutside="closeHandler"
```

Comment:
we can use a hyphenated name here like `v-click-outside`

Reply:
changed